addons/point_of_sale/static/src/app/screens/receipt_screen/receipt_screen.xml

```diff
@@ -20,17 +20,22 @@
                                     <i t-attf-class="fa {{doPrint.status === 'loading' ? 'fa-fw fa-spin fa-circle-o-notch' : 'fa-print'}} me-1" />Print Receipt
                                 </button>
                             </div>
-                            <form t-on-submit.prevent="() => doSendEmail.call()" class="send-email d-flex">
-                                <div class="input-email input-group">
-                                    <input type="email" class="form-control py-3 bg-view" t-attf-placeholder="Email: {{currentOrder.getEmailItems().join(', ')}}" t-model="state.inputEmail" />
-                                    <button class="send btn px-5" t-att-class="isValidEmail() ? 'highlight btn btn-primary' : 'btn-secondary disabled'" type="submit">
+                            <form class="send-form d-flex">
+                                <div class="input-value input-group d-flex">
+                                    <input
+                                        type="text"
+                                        class="form-control py-3 w-100 w-md-auto bg-view"
+                                        t-attf-placeholder="Write an email or phone number.."
+                                        t-model="state.inputValue"
+                                    />
+                                    <button class="send btn px-5 me-1 border border-dark mt-1 mt-md-0" t-att-class="isValidEmail() ? 'highlight btn btn-primary' : 'btn-secondary disabled'" t-on-click="() => doSendEmail.call()" type="button">
```

Comment:
the click handler here should be something like `sendReceipt`. The `sendReceipt`  will decide whether to use `email`, `sms` or who know what other ways we might have in the future. 
This way you don't need to override this button in `pos_sms`; you will just override `sendReceipt`

Reply:
![image](https://github.com/odoo/odoo/assets/127721971/28fa15b0-7e56-4ef4-8790-a06c68eb40a9)


Currently, we are displaying three buttons in a single line. It would be more effective to have separate onclick events for each button.

This approach will help streamline the handling of each button's action and improve clarity in the codebase.

Please let me know if this aligns with your expectations or if there are other considerations to discuss.